rootfs/etc/nginx/lua/util/dns.lua

```diff
@@ -0,0 +1,37 @@
+local resolver = require "resty.dns.resolver"
+
+local _M = {}
+
+function _M.resolve(host)
+  local r, err = resolver:new{
+    nameservers = {"8.8.8.8", {"8.8.4.4", 53} },
```

Comment:
We have the DNS servers to use in the controller. Maybe we can pass that during the initialization?

Reply:
Mhm - this will not work for us as it stands - we need to resolve private dns names here (split horizon dns, corporate environment)

Reply:
> Mhm - this will not work for us as it stands - we need to resolve private dns names here (split horizon dns, corporate environment)

That's why I suggested we use the inherited dns name servers the container starts

Reply:
Thanks for early feedbacks, this was just a quick copy paste from the lua-resty-dns docs, hence WIP :)

> We have the DNS servers to use in the controller. Maybe we can pass that during the initialization?

makes sense yeah, I'll do that